thoth/storages/graph/postgres.py

```diff
@@ -2894,6 +2894,60 @@ def get_python_package_version_hashes_sha256(
         )
         return [item[0] for item in query.all()]
 
+    # BOOKMARK
+    def get_python_package_required_symbols(
+        self, package_name: str, package_version: str, index_url: str = None
```

Comment:
I would probably require `index_url` to be set. Our own builds may require other symbols than the upstream ones.